flow/packet-headers/icmpv6.yaml

```diff
@@ -7,311 +7,64 @@ components:
-              pointer
-            format: integer
-            length: 32
-            default: 0
-            features: [count, metric_group]
-
-    Flow.ICMPv6EchoRequest:
-      description: Packet Header for ICMPv6 Echo Request
-      type: object
-      properties:
-        type:
-          x-field-pattern:
-            description: >-
-              type of ICMPv6 Echo Request
+              ICMPv6 echo type
             format: integer
             length: 8
             default: 128
             features: [count, metric_group]
         code:
           x-field-pattern:
             description: >-
-              code of ICMPv6 Echo Request
-            format: integer
-            length: 1
-            default: 0
-            features: [count, metric_group]
-        checksum:
-          x-field-pattern:
-            description: >-
-              checksum of ICMPv6 Echo Request
-            format: checksum
-            length: 16
-        identifier:
-          x-field-pattern:
-            description: >-
-              identifier of ICMPv6 Echo Request
-            format: integer
-            length: 16
-            default: 0
-            features: [count, metric_group]
-        sequence_number:
-          x-field-pattern:
-            description: >-
-              sequence number of ICMPv6 Echo Request
-            format: integer
-            length: 16
-            default: 0
-            features: [count, metric_group]
-
-    Flow.ICMPv6EchoReply:
-      description: Packet Header for ICMPv6 Echo Reply
-      type: object
-      properties:
-        type:
-          x-field-pattern:
-            description: >-
-              type of ICMPv6 Echo Reply
+              ICMPv6 echo sub type
             format: integer
             length: 8
-            default: 129
-            features: [count, metric_group]
-        code:
-          x-field-pattern:
-            description: >-
-              code of ICMPv6 Echo Reply
-            format: integer
-            length: 1
             default: 0
             features: [count, metric_group]
-        checksum:
-          x-field-pattern:
-            description: >-
-              checksum of ICMPv6 Echo Reply
-            format: checksum
-            length: 16
         identifier:
           x-field-pattern:
             description: >-
-              identifier of ICMPv6 Echo Reply
+              ICMPv6 echo identifier
             format: integer
             length: 16
             default: 0
             features: [count, metric_group]
         sequence_number:
           x-field-pattern:
             description: >-
-              sequence number of ICMPv6 Echo Reply
+              ICMPv6 echo sequence number
             format: integer
             length: 16
             default: 0
             features: [count, metric_group]
 
-    Flow.ICMPv6MulticastListnerQuery:
-      description: Packet Header for ICMPv6 Multicast Listner Query
+    Flow.Icmp.Common:
```

Comment:
This is the same name as in icmp.yaml - it should be changed to Flow.Icmpv6.Common

Reply:
Changed the name